src/routes/docs/tooling/appwriter/+page.markdoc

```diff
@@ -0,0 +1,55 @@
+---
+layout: article
+title: The Appwriter
+description: Learn about the custom Appwriter mechanical keyboard and its specifications
+---
+
+{% only_dark %}
+![The Appwriter](/images/docs/keyboard/dark/appwriter.png)
+{% /only_dark %}
+
+{% only_light %}
+![The Appwriter](/images/docs/keyboard/appwriter.png)
+{% /only_light %}
+
+The Appwriter is an exclusive mechanical keyboard custom-designed by the Appwrite team. It is optimized to improve developer productivity and is specially tuned to use with the Appwrite Console.
+
+The Appwriter uses icons from the Appwrite Console and Docs on specific keys, making memorizing keyboard shortcuts easier. For example, `G` then `D` is the shortcut for navigating to your project's databases, and the icon on the `D` key matches the icon for Appwrite Databases.
+
+# What's in the box {% #whats-in-the-box %}
+
+- Appwriter keyboard
+- USB-C data/charging cable
+- 2-in-1 keycap and switch puller
+- Extra PC layout keycaps
+- Information card with a QR code. 
+- Mac layout keys pre-installed for macOS (can be swapped with the included PC keycaps)
```

Comment:
This doesn't need to be a bullet because it's a description and not something else in the box. A sentence after to describe the keyboard layout is fine